### PR TITLE
BUGFIX: SD-906 only remove zoom button if it's existed

### PR DESCRIPTION
**Pull request by lvkhanh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2972
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2686

# Related Jira tasks
https://jira.intgdc.com/browse/SD-906

### CODE REVIEW

**Comment by lvkhanh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1510.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

File: src/components/core/geoChart/GeoChartRenderer.tsx [hunk 1]
```diff
@@ -170,6 +177,7 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
             this.addMapControls();
         } else {
             this.removeMapControls();
+            this.navigationControlButton = null;
```
Comment (ptquang86) [inline src/components/core/geoChart/GeoChartRenderer.tsx:180]: this line should be moved into `removeMapControls` function :)

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2698/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2940/ : SUCCESS in 26m 45s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6829/ : SUCCESS in 3m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2699/ : SUCCESS in 3s

**Comment by lvkhanh** [comment]
Successful extended test - storybook

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6830/ : SUCCESS in 3m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2700/ : SUCCESS in 3s

**Comment by naibinh** [comment]
LGTM

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3567/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2701/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2941/ : SUCCESS in 20m 27s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3568/ : SUCCESS in 15m 53s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6831/ : SUCCESS in 5m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1662/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2402/ : SUCCESS in 22m 27s

**Pull request merged**